virtcontainers/fc.go

```diff
@@ -384,8 +399,8 @@ func (fc *firecracker) fcInit(timeout int) error {
 		}
 	}()
 
-	var args []string
 	var cmd *exec.Cmd
+	args := []string{"--config-file", fc.fcConfigPath}
```

Comment:
may be worth to validate version greater than 0.19 before starting fc, what do you think?

Reply:
yep. you're right. should be more careful~~😋